app/javascript/components/GuitarGallery.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { render, screen /*, waitForElement*/ } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import axios from 'axios';
+import GuitarGallery from './GuitarGallery';
+import { guitarsUrl } from './constants';
+
+test("show loader when it's fetching data, then render Guitars", async () => {
+  axios.get.mockResolvedValueOnce({
+    data: [
```

Comment:
Can you move the whole mock data object into a fixtures file and import it from there?
This way it looks cleaner

Reply:
Sure 👍🏼

Reply:
Maybe I misunderstood you
Have you meant exporting the function of getting the mock data from axios to another file, and just call it here?

Reply:
In the tests directory, you can add a fixtures.js file where there is an `export const data = the data structure that you are using here`

Then you can import that data here.

Reply:
I have read [here](https://dev.to/davidimoore/using-fixtures-for-testing-a-reactredux-app-with-jest--enzyme-3hd0) that we should have a __fixtures__ dir at the root dir of the app, and put the fixture there 
Do you think a simple fixtures.js file in the test dir would be sufficient? @Ron-Lavi

Reply:
That's how we do it in Foreman, adding fixtures.js file in each component's test dir

Reply:
I have added a tests dir inside the components dir and added the relevant test inside 
Resolving this conversation 👍🏼